### PR TITLE
various things

### PR DESCRIPTION
**Pull request by DavidBJaffe**
1. Look for vdj_b, vdj_t and count directories from multi pipeline.
2. Delete the _ag feature label (see "enclone help lvars") because this was in fact never supported
by cellranger.
3. Significantly increase the amount of data that is used in the clonotype joining specificity test.
4. Fix bug in tracking of false positive join logging, and tweak the logging.
5. Fix misassignment of dataset `117417` in `enclone.testdata`.
6. Change the algorithm for deciding to use a donor reference allele, so that it checks all 
donor reference alleles for all V genes having the same name as the one originally assigned to
a contig.
7. Change default `MAX_DIFFS` from `50` to `55`.  Change default `MAX_DEGRADATION` 
from `3` to `2`.  Change default `MAX_SCORE` from `1,000,000` to `500,000`.  Change default `MAX_CDR3_DIFFS` from `10` to `15`.
8. Allow up to 5 (instead of 4) amino acid differences when joining chains.  This results in less spurious chains in clonotypes.

```
EFFECT OF CLONOTYPING ALGORITHM CHANGES
STATE  FPS  MERGES
old    25   63,594
new    25   63,738
```

### CODE REVIEW

File: pages/auto/history.html [hunk 1]
```diff
@@ -64,6 +64,27 @@ <h1>History</h1>
 
 <! –– ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓ -->
 
+<p>
+<b>12/9/20</b>: 
+<ul>
+<li>Delete the <code>_ag</code> antigen feature label (which was documented at
+<a href="../../pages/auto/help.lvars.html"><code>enclone help lvars</code></a>) because this
```
Comment (wyattmcdonnell) [inline pages/auto/history.html:71]: We should also add `multi` compatibility as an update here!

File: pages/auto/history.html [hunk 2]
```diff
@@ -64,6 +64,27 @@ <h1>History</h1>
...
+<li>Change the algorithm for deciding to use a donor reference allele, so that it checks all 
+donor reference alleles for all V genes having the same name as the one originally assigned to
+a contig.  In some cases this changes results.  Note that this change comes after Cell Ranger
+5.0 and with high probability will be integrated into the next major Cell Ranger release.</li>
+<li> Change the default value of <code>MAX_DIFFS</code> from <code>50</code> to <code>55</code>.  
+Change <code>MAX_DEGRADATION</code> from <code>3</code> to <code>2</code>.  Change the default
+value for <code>MAX_SCORE</code> from <code>1,000,000</code> to <code>500,000</code>.  Change
+the default value for <code>MAX_CDR3_DIFFS</code> from <code>10</code> to <code>15</code>.
+Ditto comments about Cell Ranger.</li>
+<li>Allow up to 5 (instead of 4) amino acid differences when joining chains.</li>
```
Comment (wyattmcdonnell) [inline pages/auto/history.html:82]: Suggest: Allow up to 5 (instead of 4) CDR3 amino acid differences when joining chains.

**Review by wyattmcdonnell — APPROVED** [review]
Small addition requested on `history.html.src`, otherwise good to go!

File: pages/auto/history.html (hunk 2 above)
Comment (DavidBJaffe) [reply pages/auto/history.html:82]: I do not see the difference!

**Comment by DavidBJaffe** [comment]
Good ideas, thank you!

**Pull request merged**